### ISSUE TITLE
open-zcash-pool fork from open-ethereum-pool -  Stats API Temporarily Down  Usually it's just a temporal issue and mining is not affected.

### ISSUE THREAD

**Issue by bps1980** [sammy007/open-ethereum-pool#445]
Expected Behavior

In browser at localhost site should be up and running displaying stats. Followed all directions made a couple miner adjustments with using docker but for the most part followed it to the teeth.

Current Behavior

In browser at localhost site says:

Stats API Temporarily Down

Usually it's just a temporal issue and mining is not affected.
Possible Solution

Ports but I checked all that and everything is how it should be
Steps to Reproduce (for bugs)

    Installed Ubuntu 18.04
    Install Docker
    Run Redis Image
    Run Nginx Image
    Downloaded Go
    Upacked Go Installer
    Added Go to Path
    Clone Zcash Pool GitHub
    Go to Make Folder libs
    Installed Make Tools
    Make Project
    Outer Make Folder
    Make Project
    Install nvm
    Install Zcash Dependencies
    Install Zcash
    Added Nodes and rpc
    Config'd nginx to serve api
    Config'd example.congig.json & cp to config.json

Context

Build a private pool for my mining farm
Regression

original from fork

Your Environment

    Ubuntu 18.04
    Docker with Nginx Container and Redis
    GOlang
    NVM
    NPM
    ember-cli@2.9.1
    bower

**Comment by mikeyb** [comment]
What errors does your dev tools show in browser?  What does your config look like?  What command are you using to start things?  What output do you get when running this command?

**Comment by bps1980** [comment]
yes **mikeyb** . I got it working. I still have these following warnings or errors:

The script from “http://localhost/assets/vendor-acc2b13d7776f8c9f166a422845eaa8d.js” was loaded even though its MIME type (“text/plain”) is not a valid JavaScript MIME type.

The script from “http://localhost/assets/open-ethereum-pool-9eaa27e729f56e7bbaae49d222567951.js” was loaded even though its MIME type (“text/plain”) is not a valid JavaScript MIME type. 

The problem was the remote address was set at 127.0.0.1:8080 and had no response. So by changing this config file to point to port 8080 instead of 80 fixed that issue. See below:

cat > nginx.conf <<EOF
events {
    worker_connections  1024;
}
http {
	upstream api {
	    server 127.0.0.1:8080;
	}
	server {
	    listen  80 default_server;
	    location / {
		root /dist;
		autoindex on;
	    }
	    location /api {
		proxy_pass http://api;
	    }
	}
}
EOF

Please keep in mind I did this one a little different from direction in Readme.

**Comment by goodthebest** [comment]
@bps1980 hey could you please share how did you fix API temporarily down issue?
